modules/common/src/main/java/org/opencastproject/security/api/AuthorizationService.java

```diff
@@ -43,6 +43,17 @@ public interface AuthorizationService {
    */
   boolean hasPermission(MediaPackage mp, String action);
 
+  /**
+   * Determines whether the current user can take the specified action given the access control list.
+   *
+   * @param mp
+   *          the access control list
+   * @param action
+   *          the action (e.g. read, modify, delete)
+   * @return whether the current user has the correct privileges to take this action
+   */
+  boolean hasPermission(AccessControlList mp, String action);
```

Comment:
The AccessControlList's variable, "mp", implies mediapackage.  Consider changing it to "acl".

Reply:
Thanks, good points.